### DISCUSSION TITLE
There are outstanding issues with your GitHub account. Please contact GitHub Support for more information.

### DISCUSSION THREAD

**Discussion by NaCoLiu** [community/community#65414] [GitHub Education]
### Hi everyone,

After passing the Edu authentication, my account suddenly expired today and cannot be authenticated again. Please let me contact technical support.

```
There are outstanding issues with your GitHub account. Please contact GitHub Support for more information.
```

**Comment by JackWeiw** [comment]
I met the same problem, could u please share how you get authenticated gain?

**Reply by NaCoLiu** [reply]
use school edu email, 😔

**Reply by JackWeiw** [reply]
![23](https://github.com/community/community/assets/126441921/27d9bec0-b520-46c3-ae73-82e358075ddd)
I did use my edu email, there's still something wrong. Pressing the continue bottun just give the waring "There are outstanding issues with your GitHub account. Please contact GitHub Support for more information."

**Reply by NaCoLiu** [reply]
yep, same it.

**Reply by JackWeiw** [reply]
There is a pro badge in your profile, it seems you have got your student benefit. Could you please share how u solve that.

**Reply by naitik0009** [reply]
please help me i'm going through the same issue

**Comment by zhangchi25806** [comment]
Same here

**Comment by Marblog** [comment]
Same here
